### ISSUE TITLE
Cannot read property 'source' of undefined

### ISSUE THREAD

**Issue by arjanvlek** [angular-ui/ui-select#1440]
I have upgraded angular to 1.5.0 and am using ui-select version 0.13.1
I get the following error:
Cannot read property 'source' of undefined, at line 1598 of select.js ( `var data = $select.parserResult.source (scope, { $select : {search:''}}), //Overwrite $search`)

This is my html code:

```
<ui-select
    required
    ng-model="myController.selectedItem"
    ng-disabled="myController.contract.employee.id !== null &&
    myController.contract.employee.id !== undefined"
    theme="bootstrap">
    <ui-select-match placeholder="Zoek naar medewerker...">{{$select.selected.fullName}}</ui-select-match>
    <ui-select-choices repeat="employee in myController.employees | filter: $select.search | orderBy: 'fullName'">
        <span ng-bind-html="employee.fullName | highlight: $select.search"></span>
    </ui-select-choices>
</ui-select>
```

How can I fix this issue?

**Comment by soyuka** [comment]
https://github.com/angular-ui/ui-select/commit/0e85670ed377148ac3a7d5ef6ff5ed7c46446858

**Comment by arjanvlek** [comment]
So I have to wait until this fix gets released and then upgrade to Angular 1.5.0?

**Comment by soyuka** [comment]
you can base your dependencies on master, not advised but it should work for now:

```json
#bower.json
    "ui-select": "angular-ui/ui-select#master"
```

**Comment by aaronroberson** [comment]
This has been fixed in the v0.14.3 release.
